src/components/MenuPanel/MenuPanelFooter.js

```diff
@@ -41,9 +41,7 @@ const ConnectedAccount = ({ account, onRequestEnable }) => (
         label="Preferences"
         onClick={onRequestEnable}
       >
-        <div>
-          <span>Enable account</span>
-        </div>
```

Comment:
Not sure, but I think we don't need the `div` anymore because it was only used previously to set the `flex` styling?